neuralop/layers/attention_kernel_integral.py

```diff
@@ -0,0 +1,193 @@
+import torch
+from torch import nn
+import math
+from torch.nn.init import xavier_uniform_, zeros_
+
+
+class AttentionKernelIntegral(torch.nn.Module):
+    """
+    Kernel integral transform with attention
+    Computes \int_{Omega} k(x, y) * f(y) dy,
+    where:
+          K(x, y) = \sum_{c=1}^d \q_c(x) * \k_c(y)
+          f(y) = v(y)
+
+    Parameters
+    ----------
+    channels : int, input and output channels
+    heads : int, number of attention heads
+    dim_head : int, dimension of each attention head, determines how many function bases to use for the kernel
+    pos_dim : int, dimension of domain
+    use_pe : bool, whether to use positional encoding
+    project_query : bool, whether to project the query function with pointwise linear layer
+    """
+
+    def __init__(self,
+                 channels,
+                 heads,
+                 dim_head,
+                 pos_dim,
+                 use_pe=True,    # use positional encoding
```

Comment:
Can we make the variable names mode explicit? 
e.g. `n_heads`, `head_n_channels` (or dim is fine too), `use_positional_encoding` etc?

Reply:
Will be easier for users not familiar with the architecture

Reply:
Sure, change the variable name to make them more explicit